docs/PremiumApps.md

```diff
@@ -0,0 +1,488 @@
+# Premium App Installation Guide
+
+The Splunk Operator currently provides support for automating the installation of Enterprise Security with support for other premium apps coming in the future. This page documents the prerequisites, installation steps, and limitations of deploying premium apps using the Splunk Operator.
+
+## Enterprise Security
+
+### Prerequisites
+
+Installing Enterprise Security in a Kubernetes cluster with the Splunk Operator requires the following:
+
+* Ability to utilize the Splunk Operator [app framework](https://splunk.github.io/splunk-operator/AppFramework.html) method of installation.
+* Access to the [Splunk Enterprise Security](https://splunkbase.splunk.com/app/263/) app package.
+* Splunk Enterprise Security version 6.4.1 or 6.6.0 as Splunk Operator requires Splunk Enterprise 8.2.2 or later. For more information regarding Splunk Enterprise and Enterprise Security compatibility, see the [version compatibility matrix](https://docs.splunk.com/Documentation/VersionCompatibility/current/Matrix/CompatMatrix).
+* If installing to an Indexer Cluster, access to the corresponding Splunk_TA_ForIndexers app from the Enterprise Security package. This app can be extracted using the procedure available at [Splunk ES app installation on Indexer Cluster](https://docs.splunk.com/Documentation/ES/7.0.2/Install/InstallTechnologyAdd-ons). After the Splunk_TA_ForIndexers is available, place it in a S3 bucket and use "cluster" scope in the YAML file to push the package via the cluster manager to the indexers.  
```

Comment:
Is this comment needed here? We can move it to the Installation guide for IDXC.

Reply:
good point, IDXC already has a section, so let's use that section.